### DISCUSSION TITLE
Date as three fields

### DISCUSSION THREAD

**Discussion by davshoward** [react-hook-form/react-hook-form#5314] [Q&A]
Hi there,

I'd like to have date input as three separate fields (day, month, year) similar to the [gov.uk design system](https://design-system.service.gov.uk/components/date-input/).

What's the best way to achieve this so it could be stored as YYYY-MM-DD? Also dealing with Yup validation?

Any help welcome :-)

Cheers

**Comment by bluebill1049** [comment]
Hey there, u can build a controlled component and let controller validate and collect values. Or U can use some third party date picker library too.
